frame/bags-list/fuzzer/src/main.rs

```diff
@@ -70,13 +70,20 @@ fn main() {
 				},
 				Action::Update => {
 					let already_contains = BagsList::contains(&id);
-					BagsList::on_update(&id, vote_weight).unwrap();
 					if already_contains {
+						BagsList::on_update(&id, vote_weight).unwrap();
 						assert!(BagsList::contains(&id));
+					} else {
+						BagsList::on_update(&id, vote_weight).unwrap_err();
 					}
 				},
 				Action::Remove => {
-					BagsList::on_remove(&id).unwrap();
+					let already_contains = BagsList::contains(&id);
+					if already_contains {
+						BagsList::on_remove(&id).unwrap();
+					} else {
+						BagsList::on_remove(&id).unwrap_err();
+					}
```

Comment:
@kianenigma can you sanity check the updated logic here

Reply:
you should be able to help me double check: Both `on_update` and `on_remove` in my base PR return an error if the node does not exist, which didn't used to be the case before that. 

Seems like these changes went in your previous PR: 
https://github.com/paritytech/substrate/pull/11342/files#diff-010d9ffd43a3e48c2e844e8223d9434dd3505d537942408f03af5c7716d14fa3R333
https://github.com/paritytech/substrate/pull/11342/files#diff-8d945d41310dfefb2a974a743cc5f7c27861d6d4b43b5808783e3fa7095fc8c6R263

Reply:
Does it have to be unwrap? Can we use `expect` instead?

Reply:
its a fuzzer, not a big deal either way?